check-redis/lib/check-redis.go

```diff
@@ -126,11 +126,11 @@ func checkReachable(args []string) *checkers.Checker {
 	}
 	defer c.Close()
 
-	if version, ok := (*info)["redis_version"]; !ok {
+	version, ok := (*info)["redis_version"];
```

Comment:
Original one (`if version, ok := (*info)["redis_version"]; !ok {`) would be betteer.

ref: https://golang.org/doc/effective_go.html#if